src/app/users/user-login/user-login.component.scss

```diff
@@ -0,0 +1,4 @@
+#inner {
```

Comment:
As all the design issues will be solved using default bootstrap classes, there is no need for content inside the css at all. Remove all content.